src/Aspire.Hosting.Azure.CosmosDB/AzureCosmosDBContainerResource.cs

```diff
@@ -3,20 +3,20 @@
 
 using Aspire.Hosting.ApplicationModel;
 
-namespace Aspire.Hosting.Azure.CosmosDB;
+namespace Aspire.Hosting.Azure;
```

Comment:
Anyone have opinions on this? `AzureCosmosDBResource` is in `Aspire.Hosting`. The emulator is in `Aspire.Hosting.Azure`.

Everything else in Azure is under `ASpire.Hosting.Azure`. For example, `AzureStorageResource` under `Aspire.Hosting.Azure`. `AzureCosmosDBResource` is the only one that got stuck in the top-level namespace.

Reply:
This looks fine.